src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -132,6 +132,13 @@ class IncidentsListContainer extends React.Component<Props> {
    * Incidents list is currently at the organization level, but the link needs to
    * go down to a specific project scope.
    */
+  componentWillUpdate() {
```

Comment:
This hook method is deprecated and shouldn't be used. Instead you could do this logic in componentDidUpdate or componentDidMount.